package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mechanic",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
New minor version, right?

Reply:
Honestly haven't had to worry to worry about semver ever so no idea. 😬

Reply:
I think so since it adds a feature.